pillar/config.sls

```diff
@@ -1 +1,3 @@
 local_domain: moio
+hostname: rpi
+{%- set dnsdomainname = "michelebologna.net" %}
```

Comment:
This is in a `set` form to use dnsdomainname in other variables (e.g. https://github.com/moio/raspberry-openvpn-gateway/pull/17/commits/c53ad618bb4362d8286f427c6db0c9f8301a9d97)